packages/cascara/src/components/Dashboard/Dashboard.module.scss

```diff
@@ -4,25 +4,17 @@ $widget-background: #f0f0f0;
 
 .Dashboard {
   display: block;
-  columns: 30rem;
+  // columns: 30rem; //verify we dont need this
```

Comment:
Removing this, the main layout in the dashboard is positioned in a single column:
![image](https://user-images.githubusercontent.com/87443054/151107453-da3444b1-2ad1-4ba8-a8e9-43c461d4c594.png)

Reply:
Commented addressed pairing with @brianespinosa.